src/redux/reservationsReducer.js

```diff
@@ -0,0 +1,37 @@
+import { request } from './users/api';
+
+// action type
+const FETCH_RESERVATIONS = 'reservationsReducer/FETCH_RESERVATIONS';
+const CREATE_RESERVATION = 'reservationsReducer/CREATE_RESERVATION';
+
+// initial state
+const initialState = [];
+
+// action creator
+export function fetchReservationsSuccess(data) {
+  return {
+    type: FETCH_RESERVATIONS,
+    payload: data,
+  };
+}
+
+export const createReservation = (data) => async (dispatch) => {
+  console.log(data);
```

Comment:
- [ ] Please, remove this old debugging `console.log` statement in order to improve the cleanliness of the repo.